discord/appinfo.py

```diff
@@ -232,6 +248,142 @@ def flags(self) -> ApplicationFlags:
         """
         return ApplicationFlags._from_value(self._flags)
 
+    async def edit(
+        self,
+        *,
+        reason: Optional[str] = MISSING,
+        custom_install_url: Optional[str] = MISSING,
+        description: Optional[str] = MISSING,
+        role_connections_verification_url: Optional[str] = MISSING,
+        install_params_scopes: Optional[List[str]] = MISSING,
+        install_params_permissions: Optional[Permissions] = MISSING,
+        flags: Optional[ApplicationFlags] = MISSING,
+        icon: Optional[bytes] = MISSING,
+        cover_image: Optional[bytes] = MISSING,
+        interactions_endpoint_url: Optional[str] = MISSING,
+        tags: Optional[List[str]] = MISSING,
+    ) -> AppInfo:
+        r"""|coro|
+
+        Edits the application info.
+
+        .. versionadded:: 2.4
+
+        .. note::
+
+            All parameters can be ``None`` to remove the respective one.
```

Comment:
```suggestion
        Edits the application info.

        .. versionadded:: 2.4
```

It's better to say this in every parameter's documentation. I know it's repetitive but it's harder to miss, plus doesn't break when a new parameter is added that isn't optional.